Tropos/TRCourierClient.swift

```diff
@@ -0,0 +1,13 @@
+import Courier
+
+@objc class TRCourierClient: NSObject {
```

Comment:
Elsewhere I've been using the un-prefixed name for Swift, and then only prefixing the Objective-C name. What do you think about this change?

```diff
-@objc class TRCourierClient: NSObject {
+@objc(TRCourierClient) final class CourierController: NSObject {
```